docs/modules/ROOT/pages/vshn-managed/redis/backup.adoc

```diff
@@ -20,6 +20,29 @@ spec:
 ----
 <1> The schedules for your backups.
 
+== Disable Backups
+
+To disable backups, set the `backup.enabled` field to `false`:
+
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNRedis
+metadata:
+  name: redis-app1-prod
+  namespace: <your-namespace>
+spec:
+  parameters:
+    ...
+    backup:
+      enabled: false
+    ...
+----
+
+If backups were previously enabled, existing backups will be retained until their retention period expires.
+
+WARNING: Disabling backups is meant for test instances only. It is not recommended to disable backups for production instances.
```

Comment:
I would template this part and include it in every file to avoid duplicating this for every single service